en/docs/learn/configuring-outbound-provisioning.md

```diff
@@ -91,12 +91,14 @@ Pickup now needs to set up outbound provisioning between WSO2 IS and the **Sales
     | LanguageLocaleKey                  | http://wso2.org/claims/dob                      |
     | LastName                           | http://wso2.org/claims/lastname                 |
     | LocaleSidKey                       | http://wso2.org/claims/primaryChallengeQuestion |
-    | ProfileId                          | http://wso2.org/claims/role                     |
+    | ProfileId                          | None                     |
```

Comment:
Do we need this None mapping

Reply:
yes, we should not map any local claim to profileId. we should keep it blank

Reply:
then shall we remove the mapping

Reply:
If we remove the mapping we can't add the ProfileId in `Advanced Claim Configuration`.